src/main/java/com/rackspace/ceres/app/downsample/AggregatedValueSet.java

```diff
@@ -16,15 +16,16 @@
 
 package com.rackspace.ceres.app.downsample;
 
-import java.time.Duration;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 
+import java.time.Duration;
+
 @Data
 @EqualsAndHashCode(callSuper = true)
 public class AggregatedValueSet extends ValueSet {
   double min = Double.MAX_VALUE;
-  double max = Double.MIN_VALUE;
+  double max = -Double.MAX_VALUE;
```

Comment:
I believe this is to fix Double.MIN_VALUE being used when metric value is 0 (Double.MIN_VALUE > 0) ?